lib/helpers.bash

```diff
@@ -95,7 +95,7 @@ alias reload_plugins="$(_make_reload_alias plugin plugins)"
 bash-it ()
 {
     about 'Bash-it help and maintenance'
-    param '1: verb [one of: help | show | enable | disable | migrate | update | search | version | reload | restart | doctor ] '
+    param '1: verb [one of: help | show | enable | disable | migrate | update | search | preview | version | reload | restart | doctor ] '
```

Comment:
you should add this to bash-it completion as well!

Reply:
ok

Reply:
Done and doen